openfisca_aotearoa/variables/demographics/students.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from openfisca_core.model_api import *
+from openfisca_aotearoa.entities import Person
+
+
+class is_tertiary_student(Variable):
+    value_type = bool
+    entity = Person
+    label = u"a tertiary student"
+    reference = """
+    tertiary provider means a university, polytechnic, college of education, or
```

Comment:
I agree with moving them out of the global variables space since their definition is lacking. In reflection I don't think it warrants an entity either - so lets just use them as flags for now with the student_allowance name space.

Reply:
moved